API/API/Program.cs

```diff
@@ -13,16 +15,24 @@
 builder.Services.AddEndpointsApiExplorer();
 builder.Services.AddSwaggerGen();
 
-var connectionString = builder.Configuration.GetConnectionString("PostgreSQLConnection");
+//Conection to data base
+string? connectionString = builder.Configuration.GetConnectionString("PostgreSQLConnection");
 builder.Services.AddDbContext<ProductContext>(options => options.UseNpgsql(connectionString));
 
-var app = builder.Build();
+/// <summary>
+/// This method gets called by the runtime. Use this method to add services to the container.
+/// </summary>
+
+builder.Services.AddScoped<IProductService, ProductService>();
+builder.Services.AddScoped<IProductRepository, ProductRepository>();
```

Comment:
Use methods DependencyInjectionConfigure.cs instead of.

Reply:
Done